readme.md

```diff
@@ -63,8 +63,8 @@ interface PubSubSession {
     /** close the session and stop receiving further messages */
     close: () => Promise<void>
     
-    /** ID of the key agreement key of the temporary DID the extension will use to encrypt the session messages */
-    encryptionKeyId: string
+    /** URI of the key agreement key of the temporary DID the extension will use to encrypt the session messages */
+    encryptionKeyUri: string
```

Comment:
This breaking change will go into the next version of the specification, please revert it here